app/views/content/train-to-be-a-teacher/how-to-apply-for-teacher-training.md

```diff
@@ -93,59 +93,35 @@ You can choose up to 3 teacher training courses for each application.
 
 If you do not find what you were looking for, there are [other ways to train](/train-to-be-a-teacher).
 
-Before you start, it’s particularly important to think about your referees and
-[teacher training personal statement](/train-to-be-a-teacher/how-to-apply-for-teacher-training#writing-your-teacher-training-personal-statement).
+Before you start, it’s particularly important to think about your [teacher training personal statement](/train-to-be-a-teacher/how-to-apply-for-teacher-training#writing-your-teacher-training-personal-statement).
 
 Some people do not get on the course they want because their application has
 not convinced the admissions team that they’re right for it.
 
 [Talk to an adviser for help preparing your application](/tta-service).
 
-## Choose your referees
+## Choose your references
 
-You need 2 references to apply.
+You need to provide the details of 2 people who can give you a reference when you apply. They will not be contacted until you accept a place on a course.
 
 Unlike a standard work reference, your references for teacher training need to
-be named individuals rather than (for example) a human resources team or
+be from named individuals rather than (for example) a human resources team or
 academic department.
 
-Your referees will be asked to write up to 500 words about your character and
-potential to teach.
+They'll confirm how they know you and if they know any reason why you should not work with children.
 
-It’s worth contacting them before you apply so they understand why you’re applying and what they’ll be asked to do.
+You'll need different kinds of references depending on your circumstances. If you:
 
-Choose referees who could comment on things like your:
+* are still studying or graduated in the last 5 years, you'll need an academic reference
+* already work in a school, you'll need a reference from your headteacher
 
-* communication skills
-* reliability and professionalism
-* ability to work with children
-* transferable skills
-* academic skills
+You can give a character reference, such as a mentor or someone you know from volunteering, as a second reference.
```

Comment:
```suggestion
You can give a character reference, such as from a mentor or someone you know through volunteering, as a second reference.
```

Reply:
I appreciate that we are saying references rather than referees but wonder if this sounds like odd in this sentence as it stands?